command/games/quiz_example.json

```diff
@@ -17,11 +17,11 @@
       "category": "Math",
       "type": "multiple",
       "difficulty": "easy",
-      "question": "Whats 1+4?",
+      "question": "What's 1+4?",
```

Comment:
looks like the test is not adapted yet :)

Reply:
my bad, I changed this one at the last minute without launching the tests.  :man_facepalming: 

I will fix it